service/collector/chart_config_resource.go

```diff
@@ -117,9 +118,14 @@ func (a *ChartConfigResource) Describe(ch chan<- *prometheus.Desc) error {
 }
 
 func convertToTime(datetime string) (time.Time, error) {
```

Comment:
I think these kind of functions should get unit tests when they change like that. I cannot tell if the change is legit or not and how input and how output look like.

Reply:
+1 on having unit tests for this.

Reply:
I added unittests, and I formated error logs as below. 
```
github.com/giantswarm/chart-operator/service/collector/chart_config_resource.go:130: unavailable to `2019-12-31T23:59:59Z` parsing: "parsing time \"2019-12-31T23:59:59Z\": extra text: Z"}
```